src/main/app/src/app/shared/material-form-builder/form-components/autocomplete/autocomplete.component.html

```diff
@@ -13,22 +13,48 @@
     type="text"
     [maxlength]="data.maxlength"
   />
-  <mat-icon matSuffix *ngIf="isSearching()">search</mat-icon>
+  <mat-icon
+    *ngIf="!isSearching"
+    [ngClass]="{
+      error:
+        data.formControl.invalid &&
+        (data.formControl.touched || data.formControl.dirty)
+    }"
+    matSuffix
+    >search</mat-icon
+  >
+  <button
+    *ngIf="isSearching"
+    type="button"
+    mat-icon-button
+    matSuffix
+    aria-label="Maak veld leeg"
```

Comment:
Should this not be a value that is translated? I'd prefer never to use any direct text anywhere!

Reply:
True; on it

Reply:
Fixed ✅